docs/UserGuide.adoc

```diff
@@ -91,6 +91,50 @@ Expected output:
 
 *   `Entry deleted: 1. Laksa ($10.50) [Lunch][Food]`
 
+// tag::update[]
+=== Updating a Transaction’s details: `update`
+
+Format: `update i/INDEX | PREFIX_LETTER/UPDATED_DETAILS`
+
+****
+* Updates the Transaction's details at the specified `INDEX`.
+* `PREFIX_LETTER` specifies type of detail to be updated, e.g. `n/` for name.
+* `UPDATED_DETAILS` are the new details of type corresponding to `PREFIX_LETTER` that the Transaction is to be updated with.
+****
+
+Examples:
+
+*   `update i/1 | n/Curry Laksa`
+*   `update i/1 | c/11`
+
+Expected output: +
+** Assume 2nd command occurs after 1st*
+
+*   `Entry name updated: 1. [-] Curry Laksa ($10.50) [Lunch][Food] +
+Original name was: Laksa`
+*   `Entry cost updated: 1. [-] Curry Laksa ($11) [Lunch][Food] +
+Original cost was: $10.50`
+// end::update[]
+
+// tag::clone[]
+=== Cloning a Transaction: `clone`
+
+Format: `clone i/INDEX`
+
+****
+* Clones (creates a duplicate of) a Transaction.
+* Clone produced is added to a new index at the bottom of the list.
+****
```

Comment:
1. Adds in what the `INDEX` referring to? index number that is currently shown in the UI?
2. States that INDEX must be a positive integer.

If the entire user guide `INDEX` refers to the index number that is currently shown in the UI. We can add it somewhere at the top to indicate that it applied to the entire user guide.

Reply:
Good simplification, if this is the case after everyone has completed the UserGuide then we can do this modification.